components/jupyter-web-app/backend/kubeflow_jupyter/common/utils.py

```diff
@@ -1,16 +1,23 @@
 import logging
+import os
 import sys
 import yaml
 import json
+from flask import request
 from kubernetes import client
 import datetime as dt
 from . import api
 
+# The backend will send the first config it will successfully load
 CONFIGS = [
     "/etc/config/spawner_ui_config.yaml",
     "./kubeflow_jupyter/common/yaml/spawner_ui_config.yaml"
 ]
 
+# The values of the headers to look for the User info
+USER_HEADER = os.getenv("USERID_HEADER", "X-Goog-Authenticated-User-Email")
+USER_PREFIX = os.getenv("USERID_PREFIX", "")
```

Comment:
Default prefix for IAP is `accounts.google.com:`

Reply:
done.